topi/python/topi/generic/nn.py

```diff
@@ -229,3 +229,54 @@ def schedule_binary_dense(outs):
         The computation schedule for the op.
     """
     return _default_schedule(outs, False)
+
+@tvm.target.generic_func
+def schedule_shortcut(outs):
+    """Schedule for shortcut
+
+    Parameters
+    ----------
+    outs: Array of Tensor
+      The computation graph description of shortcut
+      in the format of an array of tensors.
+
+    Returns
+    -------
+    s: Schedule
+      The computation schedule for the op.
+    """
+    return _default_schedule(outs, False)
+
+@tvm.target.generic_func
+def schedule_reorg(outs):
+    """Schedule for reorg
```

Comment:
provide unit testcases to the operators introduced

Reply:
I have test cases in nnvm, i will add operator specific test cases by next week.